src/variable.jl

```diff
@@ -1,4 +1,23 @@
-import .Ops:
+module Variables
```

Comment:
It makes it a little easier to develop on since I can just re-evaluate the `Variables` module if I change any of the types in it. I also like not having too many global symbols in a single module.